Makefile.in

```diff
@@ -78,7 +78,7 @@ POST_UNINSTALL = :
 subdir = .
 DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/Makefile.am \
 	$(top_srcdir)/configure $(am__configure_deps) \
-	$(srcdir)/config.h.in AUTHORS ChangeLog NEWS README compile \
```

Comment:
It is strange that README.md does not appear. Or you had no README.md at the time you generated this file?